boa_ast/Cargo.toml

```diff
@@ -13,10 +13,13 @@ rust-version.workspace = true
 [features]
 serde = ["boa_interner/serde", "dep:serde"]
 
+fuzzer-not-safe-for-production = ["arbitrary", "boa_interner/fuzzer-not-safe-for-production", "num-bigint/arbitrary"]
```

Comment:
I think we can change this feature to something simpler like `fuzz`. We can clarify on the docs that the fuzzer is not production-ready.

Reply:
This is more of a "traditional" feature gate name. Chromium uses something similar ("FUZZER_NOT_SAFE_FOR_PRODUCTION"). I've changed it to fuzz, it may be worthwhile for other folks to use the fuzz features.

Reply:
Completed.